src/main/java/frc/robot/subsystems/drive/DriveSubsystem.java

```diff
@@ -445,11 +449,17 @@ public void addVisionMeasurement(Pose2d pose, double timestampSeconds,
                 // (negative? before robot boot?) that then causes the poseEstimator to try to
                 // replay odometry measurements that it doesn't have. This try-catch fixes the
                 // issue, and who wants vision updates to crash their robot code anyway?
-
-                if (timestampSeconds <= 0) {
-                        return;
+                try {
```

Comment:
In this try with nested if block, I think 

hasFirstVisionMeasurement should be set to true, not false